core/services/helpers_test.go

```diff
@@ -0,0 +1,71 @@
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func ExportedSetCheckerFactory(fm FluxMonitor, fac DeviationCheckerFactory) {
+	impl := fm.(*concreteFluxMonitor)
+	impl.checkerFactory = fac
+}
+
+func mustReadFile(t testing.TB, file string) string {
+	t.Helper()
+
+	content, err := ioutil.ReadFile(file)
+	require.NoError(t, err)
+	return string(content)
+}
```

Comment:
We have cltest.MustReadFile(..) []byte, maybe we should use this instead?

Reply:
We can't use `cltest` inside `helpers_test` because it will make an import cycle. The top of this file, helpers_test.go, `import services` which is something cltest also imports.